cmd/fs/helpers_test.go

```diff
@@ -60,3 +69,219 @@ func TestFilerForWindowsLocalPaths(t *testing.T) {
 	testWindowsFilerForPath(t, ctx, `d:\abc`)
 	testWindowsFilerForPath(t, ctx, `f:\abc\ef`)
 }
+
+func mockCurrentUserApi(m *mocks.MockWorkspaceClient, u *iam.User, e error) {
+	currentUserApi := m.GetMockCurrentUserAPI()
+	currentUserApi.EXPECT().Me(mock.AnythingOfType("*context.valueCtx")).Return(u, e)
+}
+
+func mockMustWorkspaceClientFunc(cmd *cobra.Command, args []string) error {
+	return nil
+}
+
+func setupValidArgsFunctionTest(t *testing.T) (*mocks.MockWorkspaceClient, *cobra.Command) {
+	m := mocks.NewMockWorkspaceClient(t)
+	ctx := context.Background()
+	ctx = root.SetWorkspaceClient(ctx, m.WorkspaceClient)
+
+	cmd := &cobra.Command{}
+	cmd.SetContext(ctx)
+
+	return m, cmd
+}
+
+func TestGetValidArgsFunctionCompletion(t *testing.T) {
```

Comment:
Can we please also add simple integrations test for this, setting up a file tree in DBFS and ensuring the paths are read / proper APIs are called? Just to ensure the remote filer works e2e.

You can find example integration tests in `fs_cp_test.go`.

Reply:
Will add!